WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -372,6 +375,13 @@ final class EditableOrderViewModel: ObservableObject {
         return itemsInOrder
     }
 
+    /// Clears selected products and variations
+    ///
+    func clearSelectedItems() {
```

Comment:
Nit: this method should be private. Or should we move the implementation to the content of `onClearedSelection` since it's simple and easier to read there?

Reply:
Right! I made it private just to keep it separate, here: d14c167cb76de59a01e7b04b98c2931aafc1292d